superset/examples/bart_lines.py

```diff
@@ -34,7 +34,7 @@ def load_bart_lines(only_metadata: bool = False, force: bool = False) -> None:
     table_exists = database.has_table_by_name(tbl_name)
 
     if not only_metadata and (not table_exists or force):
-        content = get_example_data("bart-lines.json.gz")
+        content = get_example_data("bart-lines.json.gz", make_bytes=True)
```

Comment:
Can you give some context on why `make_bytes` is needed here?

Looking at [the docs](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_json.html) it seems that `read_json` still accepts a string. Passing `make_bytes=True` will wrap the string in a `BytesIO`, which seems unnecessary.

Reply:
@betodealmeida Because `get_example_data` returns `bytes` and pandas version(1.4.0) resulting TypeError, `make_bytes=True` make `content = BytesIO(content)` 
We can also use `StringIO` for it.
And without it we got such error:

> File "/home/runner/work/superset/superset/superset/cli/examples.py", line 113, in load_examples
    load_examples_run(load_test_data, load_big_data, only_metadata, force)
  File "/home/runner/work/superset/superset/superset/cli/examples.py", line 49, in load_examples_run
    examples.load_world_bank_health_n_pop(only_metadata, force)
  File "/home/runner/work/superset/superset/superset/examples/world_bank.py", line [60](https://github.com/apache/superset/runs/7710893773?check_suite_focus=true#step:8:61), in load_world_bank_health_n_pop
    pdf = pd.read_json(data)
TypeError: Expected file path name or file-like object, got <class 'bytes'> type
Error: Process completed with exit code 1.

![image](https://user-images.githubusercontent.com/29536522/183571826-c7e85066-3b2d-4de7-b8d9-521b5f774af7.png)

Reply:
@betodealmeida I've converted `bytes` to `string` using `decode("utf-8")`